client/web/compose/src/views/Admin/Modules/Edit.vue

```diff
@@ -340,10 +340,9 @@
               </b-tab>
 
               <b-tab
-                v-if="module.config.recordDeDup.enabled"
-                :title="$t('edit.config.validation.title')"
+                :title="$t('edit.config.uniqueValues.title')"
```

Comment:
Do we still need the old v-if here? I see you removed it, based on Vivkes feedback or?

Reply:
the `enabled` is removed

Reply:
Yes, so this is based on Vivek feedback, the `enabled` is removed